gameMechanics.js

```diff
@@ -101,4 +107,21 @@ var testInsertion = function (currentGrid, selectedColumn, playerFlag) {
 	document.write("After:<br>");
 	var newGrid = insertDisc(currentGrid, selectedColumn, playerFlag);
 	printGrid(newGrid);
+}
+
+var testButton = function (currentGrid, selectedColumn, playerFlag) {
+	var newGrid = insertDisc(currentGrid, selectedColumn,playerFlag);
+	testPrint(newGrid);
+}
+
+var testPrint = function(inputGrid) {
```

Comment:
If you are looking for a method to change the contents of the grid without refreshing the entire page, you could draw a bunch of container elements to represent the grid, then use Javascript to change the contents of specific elements.

Alternatively, you can redraw the entire grid every turn.  It's much easier to code but less performant.